src/App.js

```diff
@@ -53,7 +53,7 @@ class App extends Component {
     highlightMarker: null,
     highlightCluster: null,
     isLoading: false,
-    open: true
+    dashBoardOpen: true
```

Comment:
The entire application is a "dashboard", where the thing that opens and closes is a "side panel". Suggest you change this for clarity